src/System.Private.CoreLib/src/System/AppDomain.cs

```diff
@@ -28,9 +28,7 @@ internal sealed class AppDomain
         // of these fields cannot be changed without changing the layout in
         // the EE- AppDomainBaseObject in this case)
 
-        private AppDomainManager _domainManager;
         private Dictionary<string, object> _LocalStore;
-        private AppDomainSetup _FusionStore;
         public event AssemblyLoadEventHandler AssemblyLoad;
 
```

Comment:
There's a comment in EventSource.cs:5683 about this variable... might be worth updating it.